explorer/lib/explorer_web/live/pages/home/index.ex

```diff
@@ -1,7 +1,23 @@
 defmodule ExplorerWeb.Home.Index do
   require Logger
+  import ExplorerWeb.ChartComponents
   use ExplorerWeb, :live_view
 
+  def get_cost_per_proof_chart_data() do
+    data = Batches.get_fee_per_proofs_of_last_n_batches(100)
+
+    %{
+      data:
+        Enum.map(data, fn {fee_per_proof, _} ->
+          case EthConverter.wei_to_usd(fee_per_proof) do
+            {:ok, value} -> value
+            {:error, _} -> 0
```

Comment:
we had this `error -> 0` once before and it is not a good idea. a nil would be better

Reply:
I am afraid `nil` might break the liveview somewhere and throw a 500

Reply:
I guess the better alternative is to filter them

Reply:
Solved in [7165b55](https://github.com/yetanotherco/aligned_layer/pull/1635/commits/7165b5526d079dc6f84e86ac3f0b7552ca18fc28)